adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostSingleton.m

```diff
@@ -203,6 +208,26 @@ - (void)presentInterstitialAdForDelegate:
   [Chartboost showInterstitial:[adapterDelegate getAdLocation]];
 }
 
+#pragma mark - Banner methods
+
+- (void)configureBannerWithSize:(GADAdSize)adSize
+                       location:(nonnull NSString *)location
+                       delegate:(nullable id<CHBBannerDelegate>)delegate
+                 viewController:(nullable UIViewController *)viewController
+                         extras:(nullable GADMChartboostExtras *)extras
+{
+    if (extras.frameworkVersion && extras.framework) {
+        [Chartboost setFramework:extras.framework
+                     withVersion:extras.frameworkVersion];
+    }
+    
+    CHBBanner *banner = [[CHBBanner alloc] initWithSize:adSize.size location:location delegate:self];
+    banner.automaticallyRefreshesContent = NO;
+    [self.loadingBanners addObject:banner];
+    [self.bannersToDelegates setObject:delegate forKey:banner];
+    [banner showFromViewController:viewController];
```

Comment:
Is it necessary to call this before the Banner Ad has been loaded?

Reply:
I think so. 

Chartboost banners can be cached (loaded) and shown. 
The first method just downloads the required data without actually displaying the ad. The second one, showFromViewController:, is the counterpart to GADBannerView loadRequest:.

Here Google's SDK is requesting that the adapter loads a banner.

Reply:
Yes, -loadRequest: is when the Ad Request is sent using the Google Mobile Ads SDK.